docs/standard/microservices-architecture/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md

```diff
@@ -128,4 +128,4 @@ For further information check the article on [Team Development with Azure Dev Sp
 
 >[!div class="step-by-step"]
 >[Previous](resilient-high-availability-microservices.md)
->[Next](docker-application-development-process.md)
+>[Next](using-azure-service-fabric.md)
```

Comment:
Maybe their intention was to link to docker-application-development-process/index.md but that change seems wrong anyway since the next page continues to be using-azure-service-fabric.md. 

@ardalis please confirm.

Reply:
Goal is to remove using-azure-service-fabric.md from the sequence - if there are still links leading to it, they should be fixed (to skip it) rather than putting links to it.

Reply:
But that topic is still the next in sequence in the TOC. Why skip it?

Reply:
It shouldn't be. Just fixed in a separate PR at #12683. Waiting for confirmation from Nish then that one should be merged and this one can be closed.